src/app/components/Input/InputRange.tsx

```diff
@@ -45,7 +41,7 @@ export const InputRange = React.forwardRef<HTMLInputElement, Props>(
 					}}
 					magnitude={magnitude}
 					type="text"
-					value={values[0]}
+					value={rangeValues[0]}
```

Comment:
Just leave `values[0]` here, if the user sets above the `max` a validation message should appear (handled by `useForm`) and not change the value he entered. This is how the `input[type=number]` works.

Reply:
done @luciorubeens :+1: